### DISCUSSION TITLE
Trying Nebula Logger out and Flow is not creating log entry

### DISCUSSION THREAD

**Discussion by nikhilspacerao** [jongpie/NebulaLogger#675] [Q&A] [answered]
Hi, I'm trying to test Nebula Logger using an extremely simple Flow in a dev org, whenever an account is created log an entry to the record:
![2024-04-20 16_36_30-test - V4](https://github.com/jongpie/NebulaLogger/assets/147355187/00c7bd10-061f-4c54-99c0-c811dc49e949)
![image](https://github.com/jongpie/NebulaLogger/assets/147355187/9cef06f7-d1e5-40f9-a930-611e97495ea2)

However, I do not see any Log Entry records being created. What am I doing wrong?

**Comment by jongpie** [comment]
Hi @nikhilspacerao - you'll need to set the property "(Optional) Save Log" to "true" to create the log entries. Nebula Logger does not automatically save the log entries because Salesforce has per-transaction limits for saving (similar to how there are per-transaction limits for DML), so you'll have to specify when the logging data should be saved.

Let me know if that works for you, or if you have any other questions!

**Reply by nikhilspacerao** [reply]
@jongpie I tried now, and I am getting an error when trying to trigger the Flow. See screenshots:
![image](https://github.com/jongpie/NebulaLogger/assets/147355187/f1a661c8-3c6d-46bb-a3e5-df5ae029c245)
![image](https://github.com/jongpie/NebulaLogger/assets/147355187/5b5cedd4-d9cb-4e02-9789-62eac4a28931)

**Reply by Swizzle4372** [reply, marked as answer]
> @jongpie I tried now, and I am getting an error when trying to trigger the Flow. See screenshots: ![image](https://private-user-images.githubusercontent.com/147355187/324474788-f1a661c8-3c6d-46bb-a3e5-df5ae029c245.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2MTQ1MjIsIm5iZiI6MTcxODYxNDIyMiwicGF0aCI6Ii8xNDczNTUxODcvMzI0NDc0Nzg4LWYxYTY2MWM4LTNjNmQtNDZiYi1hM2U1LWRmNWFlMDI5YzI0NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxN1QwODUwMjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMmZjNjVmMDEwYjIwYmJkNmMwOTAwMzJjYzcxNjM3ODVhYTY0NWE3YzA0NjA2ZjIzZWI2MzFkNjViNjBjMWM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.xnsgSoEK4ZJAueSlORwGJUscOZch-QX1B5lS6vfQnig) ![image](https://private-user-images.githubusercontent.com/147355187/324474849-5b5cedd4-d9cb-4e02-9789-62eac4a28931.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2MTQ1MjIsIm5iZiI6MTcxODYxNDIyMiwicGF0aCI6Ii8xNDczNTUxODcvMzI0NDc0ODQ5LTViNWNlZGQ0LWQ5Y2ItNGUwMi05Nzg5LTYyZWFjNGEyODkzMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxN1QwODUwMjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yM2U1NDM4MDQyZDUwMGFhOTQ1M2JiNTMwZDZiZjhiZTMwM2EwOTRlYTI2ODc3NTdiYmQwOGJjNDUyNDdmYzEzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.CgNfJAmTpps0uHHXkJuswUAekH7IHTQqnekAN3cTXro)

This looks to me like it's caused by you having "true" in the Save Method field.  Save Log should be true, but if you're setting the method you'll need to use one of the provided ENUMS https://github.com/jongpie/NebulaLogger#:~:text=Enum%20Logger.SaveMethod,save%20method%20cautiously.

**Reply by jongpie** [reply]
@ministe2003 that's a great call, I had overlooked that in @nikhilspacerao's screenshot.

@nikhilspacerao based on your screenshot, if you clear/exclude the property "(Optional) Save Method)" (circled in screenshot below) and instead set the property "(Optional) Save Log" to use `{!$GlobalConstant.True}` (arrow in the screenshot below) , then I think it should work.

![image](https://github.com/jongpie/NebulaLogger/assets/1267157/e6a7ec80-9bdf-47b2-a2b8-65f5e170b3c9)

**Comment by pinoybug** [comment]
I have the same issue. Posted a separate issue
